pyincore/dataservice.py

```diff
@@ -31,12 +31,27 @@ class DataService:
 
     def __init__(self, client: IncoreClient):
         self.client = client
-        self.base_url = urljoin(client.service_url, "data/api/datasets/")
-        self.files_url = urljoin(client.service_url, "data/api/files/")
-        self.base_earthquake_url = urljoin(
-            client.service_url, "hazard/api/earthquakes/"
-        )
-        self.base_tornado_url = urljoin(client.service_url, "hazard/api/tornadoes/")
+
+        if self.client.internal:
```

Comment:
In the description, it says "when "internal" flag is set to true." How to set the internal flag?

Reply:
You can pass in the flag when instantiate the client
`client = IncoreClient(internal=True)`

Reply:
But I don't think you can test it since the network name is internal too

Reply:
code looks okay to me, but if I can't test, should I just approve it?